tools/build_rules/verifier_test/cc_indexer_test.bzl

```diff
@@ -211,10 +211,18 @@ _KytheProtoInfo = provider()
 
 def _cc_kythe_proto_library_aspect_impl(target, ctx):
     sources = _generate_files(ctx, target[ProtoInfo].direct_sources, ".pb.cc")
-    headers = _generate_files(ctx, target[ProtoInfo].direct_sources, ".pb.h")
+    if ctx.attr.enable_proto_static_reflection:
+        headers = (_generate_files(ctx, target[ProtoInfo].direct_sources, ".pb.h") +
+                   _generate_files(ctx, target[ProtoInfo].direct_sources, ".proto.h") +
+                   _generate_files(ctx, target[ProtoInfo].direct_sources, ".proto.static_reflection.h"))
```

Comment:
It might work better (and be easier to import) to change `_generate_files` to take a list of extensions rather than a single one.

Reply:
Done.